spec/src/main/doc/javase/scopescontext_se.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[contexts_se]]
+
+== Scopes and contexts in Java SE
+
+[[builtin_contexts_se]]
+
+=== Context management for built-in scopes in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
+
+
+[[request_context_se]]
+
+==== Request context lifecycle in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
+
+The request scope is active:
+
+* during any method invocation.
```

Comment:
So request context isn't shared between method invocations? I really cannot tell from the proposal as the wording is very vague. Has any thought been given on how this is supposed to be implemented?